src/Cogworks.CogFlare.Core/Components/ExternalCachePurgeComponent.cs

```diff
@@ -1,126 +1,43 @@
-﻿using RelationTypes = Umbraco.Cms.Core.Constants.Conventions.RelationTypes;
-
-namespace Cogworks.CogFlare.Core.Components;
+﻿namespace Cogworks.CogFlare.Core.Components;
 
 public class ExternalCachePurgeComponent : 
     INotificationAsyncHandler<ContentPublishedNotification>,
     INotificationAsyncHandler<ContentDeletedNotification>,
     INotificationAsyncHandler<ContentUnpublishingNotification>,
     INotificationAsyncHandler<MediaSavedNotification>
 {
-    private readonly ICloudFlareCachePurgeService _cloudFlareCachePurgeService;
-    private readonly IUmbracoContentNodeService _umbracoContentNodeService;
-    private readonly IRelationService _relationService;
-    private readonly CogFlareSettings _cogFlareSettings;
-    private readonly ILogger<ExternalCachePurgeComponent> _logger;
+    private readonly ICachePurgeService _cachePurgeService;
 
-    public ExternalCachePurgeComponent(
-        IUmbracoContentNodeService umbracoContentNodeService,
-        IRelationService relationService,
-        CogFlareSettings cogFlareSettings,
-        ICloudFlareCachePurgeService cloudFlareCachePurgeService,
-        ILogger<ExternalCachePurgeComponent> logger)
+    public ExternalCachePurgeComponent(ICachePurgeService cachePurgeService)
```

Comment:
I believe it should be our company standard to be on the newest dotnet version and use primary constructors.

Reply:
It's necessary to update to .net8 to be able to use primary constructors and that means to update umbraco version too

Reply:
Let's do it